gulpfile.js

```diff
@@ -3,20 +3,29 @@ const fs = require("fs-extra");
 const fetch = require("node-fetch");
 const yargs = require("yargs");
 const artifactoryUpload = require("gulp-artifactory-upload");
+const source = require("vinyl-source-stream");
```

Comment:
is this missing from package.json? I have a build failure locally

Reply:
true, that's a leftover, dropped it 👍